graphrag/index/flows/create_final_community_reports.py

```diff
@@ -83,9 +84,7 @@ async def create_final_community_reports(
 
     community_reports["community"] = community_reports["community"].astype(int)
     community_reports["human_readable_id"] = community_reports["community"]
-    community_reports["id"] = community_reports["community"].apply(
-        lambda _x: str(uuid4())
-    )
+    community_reports["id"] = [str(uuid4()) for _ in range(len(community_reports))]
```

Comment:
You could use `uuid4().hex` to get the value as string instead of `str()`

it is slightly faster:

```
%%timeit
[str(uuid4()) for _ in range(10000)]
``` 
61.8 ms ± 5.91 ms per loop (mean ± std. dev. of 7 runs, 10 loops each)

```
%%timeit
[uuid4().hex for _ in range(10000)]
``` 
50.3 ms ± 3.89 ms per loop (mean ± std. dev. of 7 runs, 10 loops each)